Resources/Private/TypoScript/Root.ts2

```diff
@@ -59,7 +59,13 @@ prototype(CRON.FormBuilder:CheckBoxGroup) < prototype(CRON.FormBuilder:FormEleme
 	}
 }
 
-
+prototype(CRON.FormBuilder:CheckBoxEnhanced) < prototype(CRON.FormBuilder:FormElement) {
```

Comment:
Put the new "CheckboxEnhanced" together with it's related "CheckBox" so that it is clear what belongs together. The "CheckBoxGroup" is just a container which can contain a "CheckBox" or "CheckBoxEnhanced" as far as I could unterstand. So in the places where this is defined, order them accordingly so it's easier to grasp it:

- CheckBox
- CheckBoxEnhanced
- CheckBoxGroup